app/src/main/java/com/r5k/contacerveja/data/database/repository/drink/DrinksRepository.kt

```diff
@@ -6,8 +6,8 @@ class DrinksRepository @Inject constructor(private val drinksDao: DrinksDao) :Dr
 
     override fun isDrinksRepoEmpty(): Boolean = drinksDao.loadAll().isEmpty()
 
-    override fun insertDrink(drinks: Drink): Long  {
-        return drinksDao.insert(drinks)
+    override fun insertDrink(drinksRepo: Drink): Long  {
+        return drinksDao.insert(drinksRepo)
```

Comment:
This gives me the idea that we are adding a new interface instead of adding a new drink type.